pbf/templates/player.html

```diff
@@ -80,11 +80,7 @@
 	{% endfor %}
       {% endif %}
       {% for threshold in player.thresholds %}
-      {% if threshold.achieved %}
-      <img style="position: absolute; left: {{threshold.x}}px; top: {{threshold.y}}px;" src="{% static "pbf/green.png" %}" width="20" height="20" />
-      {% else %}
-      <img style="position: absolute; left: {{threshold.x}}px; top: {{threshold.y}}px;" src="{% static "pbf/red.svg" %}" width="20" height="20" />
-      {% endif %}
+      <img style="position: absolute; left: {{threshold.x}}px; top: {{threshold.y}}px;" src="{% if threshold.achieved %}{% static "pbf/green.png" %}{% else %}{% static "pbf/red.svg" %}{% endif %}" width="20" height="20" />
```

Comment:
Is there a reason why green is a PNG and red is an SVG? 

(This is me just idly wondering -- it looks like it's not new to this change. :) )

Reply:
Interesting, isn't it? In fact there is a `green.svg` in the repository, which would almost work except for the fact that the checkmark is transparent. If someone took the time to change that, then `green.svg` would then be used. It's smaller in file size so that could be helpful, though browser caching helps.